src/libs/actions/PersistedRequests.ts

```diff
@@ -86,9 +87,13 @@ function save(requestToPersist: Request) {
     // If the command is not in the keepLastInstance array, add the new request as usual
     const requests = [...persistedRequests, requestToPersist];
     persistedRequests = requests;
-    Onyx.set(ONYXKEYS.PERSISTED_REQUESTS, requests).then(() => {
-        Log.info(`[SequentialQueue] '${requestToPersist.command}' command queued. Queue length is ${getLength()}`);
-    });
+    Onyx.set(ONYXKEYS.PERSISTED_REQUESTS, requests)
```

Comment:
@TMisiukiewicz Can you show locally what the logs look like? 

I worry this might be:
1. Quite big stuff to log
2. Does it expose the params sent as part of the request?